scripts/flush.py

```diff
@@ -40,17 +40,27 @@ def main(collection_ex_id):
     sample_units = sample_units.json()
 
     samples = [sample['id'] for sample in sample_units]
+    print(f'Sample units for collection exercise: {len(samples)}')
+
+    sample_chunks = []
+    for i in range(0, len(samples), 10):
+        sample_chunks.append(samples[i:i + 10])
+
+    sample_return = []
+    for sample_chunk in sample_chunks:
+        sample_chunk_return = requests.get(case_url + "sampleunitids?sampleUnitId=" + ','.join(sample_chunk),
+                                           auth=config["CASE_AUTH"])
+        sample_chunk_return.raise_for_status()
+        for sample in sample_chunk_return.json():
+            sample_return.append(sample)
 
-    sample_return = requests.get(case_url + "sampleunitids?sampleUnitId=" + ','.join(samples), auth=config["CASE_AUTH"])
-    sample_return.raise_for_status()
-    
     case_inprogress = []
-    for sample in sample_return.json():
-        if sample["caseGroup"]["collectionExerciseId"] == str(collection_ex[0]) and sample["caseGroup"][
-            "caseGroupStatus"] == "INPROGRESS":
+    for sample in sample_return:
+        if (sample["caseGroup"]["collectionExerciseId"] == str(collection_ex[0]) and sample["caseGroup"][
+            "caseGroupStatus"] == "INPROGRESS"):
```

Comment:
🎨 I would avoid continuation like this and try and keep the entire member block in tact: `sample["caseGroup"]["caseGroupStatus"]`